piwik-sdk/src/main/java/org/piwik/sdk/plugins/CustomDimensions.java

```diff
@@ -0,0 +1,22 @@
+package org.piwik.sdk.plugins;
+
+import org.piwik.sdk.TrackMe;
+
+/**
+ * This plugins allows you to track any Custom Dimensions defined in
+ * https://plugins.piwik.org/CustomDimensions
+ */
+public class CustomDimensions extends TrackMe {
+
+    /**
+     * This method sets a tracking API parameter dimension%dimensionId%=%dimensionValue%.
+     * Eg dimension1=foo or dimension2=bar.
+     * So the tracking API parameter starts with dimension followed by the set dimensionId.
+     * @return instance of CustomDimensions plugin
+     */
+    public synchronized CustomDimensions set(int dimensionId, String dimensionValue) {
```

Comment:
Good spotted.
`dimensionId` is 1+ https://github.com/piwik/plugin-CustomDimensions/blob/master/Dao/Configuration.php#L152